src/rpdk/core/contract/resource_client.py

```diff
@@ -288,22 +288,36 @@ def assert_failed(status, response):
         return error_code
 
     @staticmethod
+    # pylint: disable=R0913
     def make_request(
         desired_resource_state,
         previous_resource_state,
         region,
         account,
         partition,
+        action,
+        creds,
+        client_token,
+        callback_context=None,
         **kwargs
     ):
         return {
-            "desiredResourceState": desired_resource_state,
-            "previousResourceState": previous_resource_state,
-            "logicalResourceIdentifier": None,
-            **kwargs,
+            "requestData": {
+                "callerCredentials": creds,
+                "providerCredentials": creds,
+                "resourceProperties": desired_resource_state,
+                "previousResourceProperties": previous_resource_state,
+                "logicalResourceIdentifier": None,
+                "providerLogGroupName": "test_resource_contract_test",
+            },
             "region": region,
             "awsPartition": partition,
             "awsAccountId": account,
+            "action": action,
+            "callbackContext": callback_context,
+            "bearerToken": client_token,
+            "resourceType": "AWS::Service::ResourceName",
```

Comment:
The hard code `resourceType` looks weird.   Can we get the resource Type from schema?